### DISCUSSION TITLE
loading time too slow

### DISCUSSION THREAD

**Discussion by nmicun** [fullstackhero/blazor-wasm-boilerplate#178] [Q&A]
What is normal loading time? I deployed it with docker file on nginx, but as you can see from screenshot initial loading is very slow
![image](https://user-images.githubusercontent.com/56039142/163669597-75b6377b-5727-4534-8120-a32edcc873ac.png)

**Comment by yangar** [comment]
This issue is related to Blazor: https://medium.com/powered4-tv/why-we-chose-blazor-wasm-for-our-spa-c62524cde4a9
I expected it would be okay for an internal business application but perhaps not ready for a public facing site…

**Reply by neozhu** [reply]
What tool are you using, let me try it, or help me test this website to see how it goes, https://mudblazor-s.dotnet6.cn
